homeassistant/components/onkyo/config_flow.py

```diff
@@ -165,6 +167,50 @@ async def async_step_eiscp_discovery(
             ),
         )
 
+    async def async_step_ssdp(
+        self, discovery_info: ssdp.SsdpServiceInfo
+    ) -> ConfigFlowResult:
+        """Handle flow initialized by SSDP discovery."""
+        _LOGGER.debug("Config flow start ssdp: %s", discovery_info)
+
+        udn = discovery_info.ssdp_udn
+        if udn:
```

Comment:
```suggestion
        if udn := discovery_info.ssdp_udn:
```

A bit more compat, but its also a preference